SunEngine.Commons/Configuration/AddServices/AddStores.cs

```diff
@@ -21,5 +27,28 @@ public static void AddStores(this IServiceCollection services, DataBaseFactory d
 
             services.AddSingleton<SpamProtectionCache>();
         }
+
+        // Temporary solution
+        public static void AddCachePolicy(this IServiceCollection services)
+        {
+            services.AddScoped<ICachePolicy>(provider =>
+            {
+                var cacheOptions = provider.GetRequiredService<IOptions<CacheOptions>>();
+                if(cacheOptions == null)
+                    throw new NotFoundServiceException("Cache policy must be added after loading settings from database");
+
+                switch (cacheOptions.Value.CurrentCachePolicy)
+                {
+                    case CachePolicy.AlwaysPolicy:
+                        return new AlwaysCachePolicy();
+                    case CachePolicy.NeverPolicy:
+                        return new NeverCachePolicy();
+                    case CachePolicy.CustomPolicy:
+                        return new CustomCachePolicy();
+                    default:
+                        throw new ArgumentOutOfRangeException();
```

Comment:
Nice, it's good we have an exception in default block. I'd suggest adding a message to it, so it would be even better!